spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -557,10 +572,6 @@ The `Order` entity has an `addressZipCode` field, as well as an `address` field
 List<Order> findByAddress_zipcode(int zipCode);
 ----
 
-WARNING: Define entity properties following standard Java naming conventions for camel case, using underscore only as the last resort.
-
-In all places where entity property names can be specified other than `@Query`, `Id` is an alias for the entity property that is designated as the id. Entity property names that are used in _Query by Method Name_ must not contain reserved words.
```

Comment:
This section already had an exception for `Query` so it should have been fine.
Removing this will mean you can no longer write the following when the unique identifier attribute is `productIdNum`:

```
@Find
Optional<Product> get(String id);
```

Reply:
Yeah but `id` != `Id`.